src/components/PageSettingsModal/PageSettingsModal.jsx

```diff
@@ -147,6 +147,13 @@ export const PageSettingsModal = ({
                   >
                     <Box mb="0.75rem">
                       <FormLabel mb={0}>Page URL</FormLabel>
```

Comment:
When a user inputs an external link and clicks on "Save", it brings the user to the edit page which may be confusing (plus the content will not be shown to the site visitor). Should we disable the edit page if an external link is put into the page URL?

Reply:
Thinking of changing the type of these to something else instead! Which should fix the issue